src/style.css

```
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+}
+
+/* #region stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars .stars__star:last-child {
```

```suggestion
.stars__star:last-child {
```
There is no need to increase the specificity